tests/page-objects/default-mobile/reports/reports.wdio.page.js

```diff
@@ -13,7 +13,7 @@ const verifyMultiselectElementsDisplay = async (shouldHide=false) => {
   await (await selectedReportsCount()).waitForDisplayed({ reverse: shouldHide });
   return {
     countLabel: shouldHide ? false : await (await selectedReportsCount()).getText(),
-    selectedCount: (await reportsPageDefault.selectedReportsCheckboxes()).length,
+    selectedCount: (await reportsPageDefault.leftPanelSelectors.selectedReportsCheckboxes()).length,
   };
 };
 
```

Comment:
We can also reduce code duplication in these two followings functions `selectAll`, `DeselectAll`.
```
const toggleSelection = async (selector, reverse = false) => {
  const element = await $(selector);
  await element.waitForDisplayed();
  await element.click();
  await element.waitForDisplayed({ reverse });
  return await verifyMultiselectElementsDisplay(reverse);
};
```
And then:
const selectAll = async () => {
  return await toggleSelection(SELECT_ALL);
};

const deselectAll = async () => {
  return await toggleSelection(DESELECT_ALL, true);
};